connector/core/core-service/src/main/java/org/oaebudt/edc/core/OaebudtCoreExtension.java

```diff
@@ -15,9 +19,21 @@ public String name() {
         return NAME;
     }
 
+    @Override
+    public void initialize(ServiceExtensionContext context) {
+        ServiceExtension.super.initialize(context);
+    }
+
     @Provider(isDefault = true)
-    public ParticipantGroupStore defaultParticipantStore() {
+    public ParticipantGroupStore inMemoryParticipantStore() {
         return new InMemoryParticipantGroupStore();
     }
 
+    @Provider
+    public ParticipantGroupStore defaultParticipantStore(ServiceExtensionContext context) {
+        String dataSourceUrl = context.getSetting("web.datasource.mongo.url", "no:op");
+        MongoClient client = MongoClients.create(dataSourceUrl);
+        return new MongoParticipantGroupStore(client);
```

Comment:
Why are we using mongo to store connector relative data? at the moment we are using 2 different databases (mongo and postgres), having useless complexity.
If, as I imagine, the answer to my concern is "we don't have time we need to deliver" as I heard too often during the past weeks, I'll just avoid to review, you can just go and merge despite my opinion.

Reply:
Hi @ndr-brt 

True. We should remove all mongo implementation.
I agree. We don't have time constraint for this one. I'll go ahead and fix this. 

You opinion is always greatly valued. 🙇🏽